src/tests/resistance/fuzzing_flood.rs

```diff
@@ -0,0 +1,640 @@
+
+    let mut request_table = Table::new();
+    request_table
+        .load_preset(UTF8_FULL)
+        .apply_modifier(UTF8_ROUND_CORNERS)
+        .set_header(vec![
+            Cell::new("Peers").set_alignment(CellAlignment::Center),
+            Cell::new("Requests").set_alignment(CellAlignment::Center),
+            Cell::new("Min (ms)").set_alignment(CellAlignment::Center),
+            Cell::new("Max (ms)").set_alignment(CellAlignment::Center),
+            Cell::new("stddev (ms)").set_alignment(CellAlignment::Center),
+            Cell::new("10% (ms)").set_alignment(CellAlignment::Center),
+            Cell::new("50% (ms)").set_alignment(CellAlignment::Center),
+            Cell::new("75% (ms)").set_alignment(CellAlignment::Center),
+            Cell::new("90% (ms)").set_alignment(CellAlignment::Center),
+            Cell::new("99% (ms)").set_alignment(CellAlignment::Center),
+            Cell::new("Request/s").set_alignment(CellAlignment::Center),
+        ]);
+
+    let mut stats_table = Table::new();
+    stats_table
+        .load_preset(UTF8_FULL)
+        .apply_modifier(UTF8_ROUND_CORNERS)
+        .set_header(vec![
+            Cell::new("Peers").set_alignment(CellAlignment::Center),
+            Cell::new("Requests").set_alignment(CellAlignment::Center),
+            Cell::new("Max active").set_alignment(CellAlignment::Center),
+            Cell::new("Handshake\nAccepted").set_alignment(CellAlignment::Center),
+            Cell::new("Handshake\nRejected").set_alignment(CellAlignment::Center),
+            Cell::new("Handshake\nErrors").set_alignment(CellAlignment::Center),
+            Cell::new("Connection\nDropped").set_alignment(CellAlignment::Center),
+            Cell::new("Corrupt\nTerminated").set_alignment(CellAlignment::Center),
+            Cell::new("Corrupt\nRejected").set_alignment(CellAlignment::Center),
+            Cell::new("Corrupt\nIgnored").set_alignment(CellAlignment::Center),
+            Cell::new("Corrupt\nReplied").set_alignment(CellAlignment::Center),
+            Cell::new("IO Errors").set_alignment(CellAlignment::Center),
+            Cell::new("Bad replies").set_alignment(CellAlignment::Center),
+            Cell::new("Hung\nConnection").set_alignment(CellAlignment::Center),
+            Cell::new("Time (s)").set_alignment(CellAlignment::Center),
+        ]);
+
+    // Create a pool of valid and invalid message types
+    const MAX_VALID_MESSAGES: usize = 100;
+    let peer_counts = vec![
+        1, 10, 20, 30, 40, 50, 60, 70, 80, 90, 100, 200, 300, 500, 750, 800,
+    ];
+
+    let mut rng = seeded_rng();
+    let valid_pool = valid_queries_responses();
+
+    let mut node: Node = Default::default();
+    node.initial_action(Action::SeedWithTestnetBlocks {
+        socket_addr: new_local_addr(),
+        block_count: 3,
+    })
+    .max_peers(peer_counts.iter().max().unwrap() * 2 + 10)
```

Comment:
Is this arbitrarily higher than the peer count?

Reply:
Yes - just to ensure that the node should technically accept all of peer connections.